website/static/js/anonymousLogActionsList.json

```diff
@@ -0,0 +1,77 @@
+{
+    "project_created" : "A user created a project",
+    "project_registered" : "A user registered a project",
+    "project_registered_no_user": "Project registered",
+    "prereg_registration_initiated": "A user submitted for review to the Preregistration Challenge a registration of a project",
+    "project_deleted" : "A user deleted a project",
+    "created_from" : "A user created a project based on a template",
+    "pointer_created" : "A user created a link to a project",
+    "pointer_forked" : "A user forked a link to a project",
+    "pointer_removed" : "A user removed a link to a project",
+    "made_public" : "A user made a project public",
+    "made_public_no_user" : "Project made public",
+    "made_private" : "A user made a project private",
+    "tag_added" : "A user added a tag to a project",
+    "tag_removed" : "A user removed a tag from a project",
+    "edit_title" : "A user changed the title of a project",
+    "edit_description" : "A user edited the description of a project",
+    "updated_fields" : "A user changed the ${updated_fields} for a project",
+    "external_ids_added" : "A user created external identifiers on a project",
+    "citation_added" : "A user added a citation to a project",
+    "citation_edited" : "A user updated a citation in a project",
+    "citation_removed" : "A user removed a citation in a project",
+    "contributor_added" : "A user added contributor(s) to a project",
+    "contributor_removed" : "A user removed contributor(s) from a project",
+    "contributors_reordered" : "A user reordered contributors for a project",
+    "permissions_updated" : "A user changed permissions for a project",
+    "made_contributor_visible" : "A user made contributor(s) visible on a project",
+    "made_contributor_invisible" : "A user made contributor(s) invisible on a project",
+    "wiki_updated" : "A user updated a wiki page of a project",
+    "wiki_deleted" : "A user deleted a wiki page of a project",
+    "wiki_renamed" : "A user renamed a wiki page of a project",
+    "made_wiki_public" : "A user made the wiki of a project publicly editable",
+    "made_wiki_private" : "A user made the wiki of a project privately editable",
+    "addon_added" : "A user added an addon to a project",
+    "addon_removed" : "A user removed an addon from a project",
+    "addon_file_moved" : "A user moved a file",
+    "addon_file_copied" : "A user copied a file",
+    "addon_file_renamed" : "A user renamed a file",
+    "folder_created" : "A user created a folder in a project",
+    "file_added" : "A user added a file to a project",
+    "file_updated" : "A user updated a file in a project",
+    "file_removed" : "A user removed a file from a project",
```

Comment:
The anonymized logs were demoed to @saradbowman in this way. Since the API doesn't return any information about the node type when you request with a VOL, we can either display ```a project``` for all logs, or nothing (i.e. "A user added a file" instead of "A user added a file to a project"). @saradbowman, thoughts?